unity3d/Assets/Swrve/Editor/SwrvePostProcess.cs

```diff
@@ -45,10 +50,57 @@ public static void OnPostprocessBuild(BuildTarget target, string pathToBuiltProj
             if (OPEN_WORKSPACE && !UnityEditorInternal.InternalEditorUtility.inBatchMode) {
                 Process.Start (string.Format ("file://{0}/Unity-iPhone.xcworkspace", pathToBuiltProject));
             }
-		}
-		else
-		{
-			UnityEngine.Debug.Log("post process for Android");
-		}
-	}
+    		}
+    		else
+    		{
+    			UnityEngine.Debug.Log("post process for Android");
+    		}
+  	}
+
+    private static void CorrectXCodeProject (string pathToProject, bool writeOut)
+    {
+        string path = Path.Combine (Path.Combine (pathToProject, "Unity-iPhone.xcodeproj"), "project.pbxproj");
+        string xcodeproj = File.ReadAllText (path);
+
+        xcodeproj = FixList ("OTHER_CFLAGS", xcodeproj);
+        xcodeproj = FixList ("OTHER_LDFLAGS", xcodeproj);
+        xcodeproj = FixList ("HEADER_SEARCH_PATHS", xcodeproj);
+        xcodeproj = FixList ("GCC_PREPROCESSOR_DEFINITIONS", xcodeproj);
+        xcodeproj = FixValue ("SDKROOT", xcodeproj, "\"iphoneos\"");
```

Comment:
It *specifically* fixes for the simulator.  When Unity builds for there, it sets SDKROOT to "iphonesimulator" which XCode doesn't recognise, and doesn't have a clue how to build for!